plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/messages.properties

```diff
@@ -5,3 +5,25 @@ APPENGINE_GUESTBOOK_ARCHETYPE_DESCRIPTION=Generates the guestbook demo sample, c
 APPENGINE_GUESTBOOK_ARCHETYPE_DISPLAY_NAME=Guestbook example
 APPENGINE_SKELETON_ARCHETYPE_DESCRIPTION=Generates a new, empty App Engine project ready for your own classes and resources, but with required files and directories.
 APPENGINE_SKELETON_ARCHETYPE_DISPLAY_NAME=Hello-world template
+LOCATION_GROUP_TEXT=Location
```

Comment:
In many of the other modules we use lowercase dot separated format, e.g.: `location.group.text`. Clearly not a critical issue, but an inconsistency.